src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -378,6 +378,8 @@ class OpCredsFabricTableDelegate : public chip::FabricTable::Delegate
         ChipLogProgress(Zcl, "OpCreds: Fabric index 0x%x was removed", static_cast<unsigned>(fabricIndex));
 
         EventManagement::GetInstance().FabricRemoved(fabricIndex);
+        InteractionModelEngine::GetInstance()->OnFabricRemoved(fabricIndex);
```

Comment:
Why is this here? Why isn't the IM engine directly registering itself as a FabricTable::Delegate by calling `mpFabricTable->AddFabricDelegate` at init time?

That will ensure it works consistently across both server and controller modalities.

Reply:
added, thanks